frameworks/opennmt_lua/Dockerfile

```diff
@@ -1,14 +1,60 @@
+FROM ubuntu:14.04 as opennmt_builder
+
+RUN apt-get update && apt-get install -y \
+        build-essential \
+        cmake \
+        wget \
+        gcc \
+        git \
+        libboost-python-dev \
+        python-dev
+
+RUN wget http://www.cmake.org/files/v3.2/cmake-3.2.2.tar.gz && \
+	tar xf cmake-3.2.2.tar.gz && \
+	cd cmake-3.2.2 && \
+	./configure && \
+	make && \
+	make install
+
+ARG OPENNMT_LUA_URL
+ENV OPENNMT_LUA_URL=${OPENNMT_LUA_URL:-https://github.com/OpenNMT/OpenNMT.git}
+ARG OPENNMT_LUA_REF
+ENV OPENNMT_LUA_REF=${OPENNMT_LUA_REF:-master}
+ARG OPENNMT_TOKENIZER_REF
+ENV OPENNMT_TOKENIZER_REF=${OPENNMT_TOKENIZER_REF:-master}
+
+RUN git clone --depth 1 --branch ${OPENNMT_LUA_REF} --single-branch ${OPENNMT_LUA_URL} /root/opennmt
```

Comment:
The newest `opennmt/opennmt` image already includes the OpenNMT code. Is that required?